### PR TITLE
LPS-190841 - Slow publication of no A-Z or 0-9 comments.

### PR DESCRIPTION
**Pull request by manuelRives**
https://liferay.atlassian.net/browse/LPS-190841

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java [hunk 1]
```diff
@@ -2229,6 +2229,10 @@ private String _getUniqueUrlSubject(
 
 		String uniqueUrlSubject = urlSubject;
 
+		if (uniqueUrlSubject.equals(String.valueOf(StringPool.DASH))) {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java:2232]: String.valueOf String?

Use 
`Objects.equals( urlSubject, StringPool.DASH)`

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java (hunk 1 above)
Comment (manuelRives) [reply modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java]: My bad, I was originally using CharPool.

Changed.

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
Test failures are unrelated

**Comment by austinchiang** [comment]
ci:forward:force

**Comment by austinchiang** [comment]
ci:forward:force

**Pull request closed without merging**